Sources/PointFree/Enterprise.swift

```diff
@@ -25,7 +25,6 @@ let enterpriseLandingResponse: AppMiddleware<Tuple2<User?, EnterpriseAccount.Dom
       headersMiddleware: flash(.warning, "That enterprise account does not exist.")
     )
     )
-    <<< validateMembership
```

Comment:
- [ ] Make sure subscription info is threaded through to landing page so "Subscribe" link isn't in header.